io.trezor.suite.yml

```diff
@@ -0,0 +1,72 @@
+app-id: io.trezor.suite
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: trezor-suite
+separate-locales: false
+rename-icon: trezor-suite
+finish-args:
+# Essential to get the autostart working.
+  - --filesystem=~/.config/autostart
+  - --share=network
+# Trezor USB devices can be detected only when using "all."
+  - --device=all
+#  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=x11
+#  - --socket=wayland
+# There are plans to possibly introduce sound effects in the future.
+#  - --socket=pulseaudio
+# There are plans to possibly introduce the status icon in the future.
+#  - --talk-name=org.kde.StatusNotifierWatcher
```

Comment:
can you remove the commented portions out?

Reply:
sure!